.editorconfig

```diff
@@ -0,0 +1,28 @@
+# Use editorconfig to stop whitespace wars
+# http://EditorConfig.org
+
+# top-most EditorConfig file
+root = true
+
+# Default to using tabs
+# Unix-style newlines with a newline ending every file
+[*]
+end_of_line = lf
+insert_final_newline = true
+indent_style = tab
+charset = utf-8
+
+# 2 space indent for yaml
+[*.{yml,yaml}]
+indent_style = space
+indent_size = 2
+
+# 2 space js
+[*.{js,ts,jsx,json}]
+indent_style = space
+indent_size = 2
+
+# 2 space css
+[*.{scss,css,less}]
+indent_style = space
+indent_size = 2
```

Comment:
This might have been clobbered when we merged repos.... I'll drag in the gds-way one and update it with any missing filetypes

![yes](https://img.shields.io/badge/willfix-✓-green.svg?style=flat-square)